src/SmartComponents/Actions/_actions.scss

```diff
@@ -1 +1,10 @@
-.actions__view .actions__card{ text-transform: capitalize; }
\ No newline at end of file
+@import '~@red-hat-insights/insights-frontend-components/Utilities/_variables';
+
+.page__actions.actions__view {
+    .ins-p-page-header__title { text-transform: capitalize; }
+
+    table.rules-table th:first-child { width: 50%; }
+    table.rules-table td:nth-child(n+2):nth-child(-n+6) { text-align: center; }
```

Comment:
I think this could be better written as `table.rules-table td:not(:first-of-type) { text-align: center; }`